src/Akka.Persistence.Redis/RedisExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using StackExchange.Redis;
+
+namespace Akka.Persistence.Redis
+{
+    public static class RedisExtensions
+    {
+        public static bool IsClustered(this IConnectionMultiplexer connection)
```

Comment:
So we can infer whether clustering is enabled via the connection string? Nice.

Reply:
We have to iterate the servers and check their type, actually. It's not an expensive operation since we short circuit the iteration.